glutin/src/surface.rs

```diff
@@ -519,7 +519,7 @@ pub enum RawSurface {
     #[cfg(glx_backend)]
     Glx(u64),
 
-    /// HWND
+    /// Either an `HWND` or `HPBUFFEREXT` depending on [`SurfaceType`].
```

Comment:
```suggestion
    /// Either a `HWND` or `HPBUFFEREXT` depending on [`SurfaceType`].
```

sorry, I hope that's the last one.

Reply:
This one is actually a point of contention - in American English the `h` in `haitch` is silent... thus my decision to use an here, as I'd pronounce it as "an aitch wind"... but this is not a hill I want to die on :)

Reply:
I mean, I know this rule, it's not silent here since it's an abbreviation and all syllables are pronounced as the should. `an` should be used when the `h` is like in `hour`, so not pronounced, here it's `hardware` iirc if you want to be more pedantic, but it's still pronounced.

Reply:
Aye, you're correct that's how it *should* pronounced given it's an acronym/abbreviation - but I don't think I've ever heard anyone put emphasis on an `H` when it's not followed by a vowel.

Ultimately this is kind of a `gif` vs. `jif` thing (`gif` being "correct", yet the one to coin the moniker still insists it's `jif`) so it's not worth losing any sleep over. Down to personal taste, so as I said, not a hill worth dying on today. 😛

Reply:
I think `hardware`(`hard` in general) is a good example of it, since it's not `ardware` or `ard`.

Reply:
https://learn.microsoft.com/en-us/windows/apps/develop/ui-input/retrieve-hwnd